docs/en/users/07_Frequently_Asked_Questions.md

```diff
@@ -57,3 +57,16 @@ Examples with _uBlock_:
 
 - Whitelist your FreshRSS instance by adding it in _uBlock > Open the dashboard > Whitelist_.
 - Authorize your FreshRSS instance to call `sharing` configuration page by adding the rule `*sharing,domain=~yourdomain.com` in _uBlock > Open the dashboard > My filters_
+
+## Problems with firewalls
+
+If you have the error "Blast! This feed has encountered a problem. Please verify that it is always reachable then update it.", it might be because of a firewall misconfiguration.
+
+To identify the problem, here are the steps to follow:
+
+- step 1: Try to reach the feed locally to discard a problem with the feed itself. You can use your browser to this purpose.
+- step 2: Try to reach the feed from the host in which FreshRSS is installed. Something like `time curl -v 'http://myfeed.com'` should make the deal. If you are running FreshRSS within a Docker container, then you can check connectivity from within the container itself with something similar to `sudo docker exec freshrss php -r "readfile('http://myfeed.com');"`. If none of this works, then it might be a problem with your firewall.
```

Comment:
```suggestion
- step 2: Try to reach the feed from the host in which FreshRSS is installed. Something like `time curl -v 'https://github.com/FreshRSS/FreshRSS/commits/master.atom'` should make the deal. If you are running FreshRSS within a Docker container, then you can check connectivity from within the container itself with something similar to `sudo docker exec freshrss php -r "readfile('https://github.com/FreshRSS/FreshRSS/commits/master.atom');"`. If none of this works, then it might be a problem with your firewall.
```

If you need fake addresses in documentation, use a standard such as `example.net` https://en.wikipedia.org/wiki/Example.com

Reply:
Agree. Thanks for the suggestion.